relay-event-normalization/src/eap/mod.rs

```diff
@@ -90,6 +96,43 @@ pub fn normalize_attribute_types(attributes: &mut Annotated<Attributes>) {
     }
 }
 
+/// Returns `true` if the passed array is an array we currently support.
+///
+/// Currently all arrays must be homogeneous types.
+fn is_supported_array(arr: &[Annotated<Value>]) -> bool {
```

Comment:
This function filters out arrays that aren't empty, but contain no nonempty values. Is that intentional? In other words, should the array `[null, null, null]` be rejected, given that `[3, null, null]` is accepted?

Reply:
Oh good catch, `[null, null, null]` should be accepted, as we need that for `Annotated` values with errors.